kokoro/windows/gapid.wxs

```diff
@@ -49,15 +49,39 @@
         <Shortcut Id="ApplicationStartMenuShortcut" Name="GAPID" Description="Graphics API Debugger"
             Target="[!gapid.exe]" WorkingDirectory="APPLICATIONROOTDIRECTORY" Icon="gapid.ico" />
         <RemoveFolder Id="CleanUpShortCut" Directory="ApplicationProgramsFolder" On="uninstall" />
-        <RegistryValue Root="HKCU" Key="Software\Google\GAPID" Name="installed" Type="integer" Value="1" KeyPath="yes" />
+        <RegistryValue Root="HKCU" Key="SOFTWARE\Google\GAPID" Name="installed" Type="integer" Value="1" KeyPath="yes" />
       </Component>
     </DirectoryRef>
 
     <Icon Id="gapid.ico" SourceFile="gapid.ico" />
 
+    <DirectoryRef Id="GAPID">

```

Comment:
Does this automatically delete these keys on uninstallation? I vaguely remember there needing to be some additional attribute for telling WiX that these keys are safe to delete on uninstall.

Reply:
Yes, I've tested it and the registry values all get cleaned up on uninstall.